README.md

```diff
@@ -14,7 +14,7 @@ Clone [this repository](https://github.com/aframevr/aframe-site):
 
 To install the dependencies and start the local development server:
 
-    npm install && npm run installdocs && npm start
+    npm install && npm start
```

Comment:
IME, because there's not a `postinstall` npm script, `npm run installdocs` is required (if your local clone of the repo isn't on the latest docs version). I think this should be in a `postinstall` script. I brought it up before, but it was dismissed (not sure why).

Reply:
I see below you added a [`postinstall` npm script in the `package.json`](https://github.com/aframevr/aframe-site/pull/466/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R53). I'm in favour of this.

Reply:
8012fd89395b7281ed22faf6e7dd50aba1822996